src/main/java/programminglife/parser/Cache.java

```diff
@@ -0,0 +1,245 @@
+     */
+    @NotNull
+    public String getSequence(int nodeID) {
+        if (getSequenceMap().containsKey(nodeID)) {
+            return getSequenceMap().get(nodeID);
+        } else {
+            throw new NoSuchElementException(String.format("No sequence is cached for node %d", nodeID));
+        }
+    }
+
+    /**
+     * Set the sequence for the node with NodeId.
+     * @param nodeID ID of the node to set the sequence for.
+     * @param sequence new sequence.
+     */
+    @NotNull
+    public void setSequence(int nodeID, String sequence) {
+        getSequenceMap().put(nodeID, sequence);
+        getSequenceLengthMap().put(nodeID, sequence.length());
+    }
+
+    /**
+     * Get the sequence length for the node with NodeId.
+     * @param nodeID ID of the node to get the sequence length for.
+     * @return the length of the sequence.
+     */
+    @NotNull
+    public int getSequenceLength(int nodeID) {
+        if (getSequenceLengthMap().containsKey(nodeID)) {
+            return getSequenceLengthMap().get(nodeID);
+        } else {
+            throw new NoSuchElementException(String.format("No sequence length is cached for node %d", nodeID));
+        }
+    }
+
+    /**
+     * Get the name of a {@link programminglife.model.Genome} based on its index.
+     * @param genomeID the index (0-based) of the {@link programminglife.model.Genome} in the GFA header
+     * @return the name of the {@link programminglife.model.Genome}
+     */
+    @NotNull
+    public String getGenomeName(int genomeID) {
+        if (getGenomeNamesMap().containsKey(genomeID)) {
+            return getGenomeNamesMap().get(genomeID);
+        } else {
+            throw new NoSuchElementException(String.format("No name is cached for genome %d", genomeID));
+        }
+    }
+
+    /**
+     * Add the name of a {@link programminglife.model.Genome}, index is previous one + 1.
+     * @param genomeName the name of the {@link programminglife.model.Genome} to add
+     */
+    @NotNull
+    public void addGenomeName(String genomeName) {
+        int index = getGenomeNamesMap().size();
+        getGenomeNamesMap().put(index, genomeName);
+    }
+
+    /**
+     * Completely remove a database. This cannot be undone.
+     * @return true if the file was removed, false if it did not exist
+     * @throws IOException when something strange happenes during deletion
+     */
+    public boolean removeDB() throws IOException {
+        System.out.printf("[%s] Removing database %s%n", Thread.currentThread().getName(), this.dbFileName);
+        close();
+        return Files.deleteIfExists(Paths.get(this.dbFileName));
+    }
+
+    /**
+     * Remove a cache file.
+     * @param name the name of the file to remove
+     * @return true if the file was deleted
+     * @throws IOException when strange things happen
+     */
+    public static boolean removeDB(String name) throws IOException {
+        return Files.deleteIfExists(Paths.get(toDBFile(name)));
```

Comment:
This is not possible from a static context. This method is only used to remove test files or remove an uninitialized method at startup.